From a78ac6ff00b5b1320e71711f54e951ae6114f02c Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Mon, 2 Dec 2019 15:02:04 -0600
Subject: [PATCH 32/34] ARM: dts: logicod-imx8mm-kit: Set i2c3 to 400KHz

According to the schematic cover page, the devices on i2c3 are
all capable of operating at 400KHz, but the clock is only set
to 400KHz.

This patch increases the clock speed of i2c3 to 400KHz

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/lpd-imx8mm-som.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/lpd-imx8mm-som.dtsi b/arch/arm64/boot/dts/freescale/lpd-imx8mm-som.dtsi
index 32f5d5b79fb9..24886e7ec597 100644
--- a/arch/arm64/boot/dts/freescale/lpd-imx8mm-som.dtsi
+++ b/arch/arm64/boot/dts/freescale/lpd-imx8mm-som.dtsi
@@ -187,7 +187,7 @@
 };
 
 &i2c3 {
-	clock-frequency = <100000>;
+	clock-frequency = <400000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c3>;
 	status = "okay";
-- 
2.17.1

